metadata/orm/doi_info.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""TransactionKeyValue links Transactions and Keys and Values objects."""
+from peewee import ForeignKeyField, CharField
+from metadata.orm.base import DB
+from metadata.orm.doi_entries import DOIEntries
+from metadata.rest.orm import CherryPyAPI
+from metadata.orm.utils import unicode_type
+
+
+class DOIInfo(CherryPyAPI):
+    """
+    TransactionKeyValue attributes are foreign keys.
+
+    Attributes:
+        +-------------------+-------------------------------------+
+        | Name              | Description                         |
+        +===================+=====================================+
+        | doi               | Link to the DOIEntries model        |
+        +-------------------+-------------------------------------+
+        | key               | Link to the Keys model              |
```

Comment:
These aren't links... should they be?

Reply:
Nope. Just missed renaming the stuff in the comments. =)